templates/ts/custom-copilot-rag-azure-ai-search/src/config.ts.tpl

```diff
@@ -0,0 +1,18 @@
+const config = {
+  botId: process.env.BOT_ID,
+  botPassword: process.env.BOT_PASSWORD,
+  {{#useOpenAI}}
+  openAIKey: process.env.OPENAI_API_KEY,
+  openAIModelName: "gpt-3.5-turbo",
```

Comment:
shall we also put `openAIEmbeddingModelName` here and give default value='text-embedding-ada-002'. Then in utils.ts, we refer this config.openAIEmbeddingModelName.

This is in case that user what to change another model and they can easily find where to configure it.